components/camel-hdfs/src/main/java/org/apache/camel/component/hdfs/HdfsArrayFileTypeHandler.java

```diff
@@ -36,8 +36,14 @@
             HdfsInfo hdfsInfo = hdfsInfoFactory.newHdfsInfo(hdfsPath);
             HdfsConfiguration endpointConfig = hdfsInfoFactory.getEndpointConfig();
             Class<? extends WritableComparable> valueWritableClass = endpointConfig.getValueType().getWritableClass();
-            rout = new ArrayFile.Writer(hdfsInfo.getConfiguration(), hdfsInfo.getFileSystem(), hdfsPath, valueWritableClass,
-                    endpointConfig.getCompressionType(), () -> { });
+            rout = new ArrayFile.Writer(
+                    hdfsInfo.getConfiguration(),
+                    hdfsInfo.getFileSystem(),
+                    hdfsPath,
+                    valueWritableClass,
+                    endpointConfig.getCompressionType(),
+            () -> { }
```

Comment:
nit: formatting ident here

Reply:
I get a check-style violation when I move the lambda inline with the others.

Reply:
That is weird, we can take a look at the checkstyle later